<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volcano</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="Mdi" name="mdiArea">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAfficher_les_param_tres"/>
    <addaction name="actionAfficher_Cacher_le_contr_le_de_flux"/>
    <addaction name="actionAttacher_D_tacher_une_fen_tre"/>
    <addaction name="separator"/>
    <addaction name="actionVue_Tabulation"/>
    <addaction name="actionVue_Libre"/>
    <addaction name="actionVue_par_d_faut"/>
    <addaction name="actionPlein_cran"/>
    <addaction name="separator"/>
    <addaction name="actionSauvegarder_les_donn_es_acquises"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>123</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Paramètres</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="parametersAreaSrolled">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>198</width>
          <height>411</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="parametersArea">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Contrôle du flux</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="mdiAreaMode"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPlay">
         <property name="text">
          <string>Lancer les flux</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonactiveHandle">
         <property name="text">
          <string>Arreter les traitements</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPrevious">
         <property name="text">
          <string>Précédent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNext">
         <property name="text">
          <string>Suivant</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="labelCurseur">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderCurseur">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Configurer la source 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Configurer la source 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Configurer la fin de traitement</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAfficher_les_param_tres">
   <property name="text">
    <string>Afficher/Cacher les paramètres</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionVue_Tabulation">
   <property name="text">
    <string>Vue Tabulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionVue_Libre">
   <property name="text">
    <string>Vue Libre</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionVue_par_d_faut">
   <property name="text">
    <string>Vue par défaut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAfficher_Cacher_le_contr_le_de_flux">
   <property name="text">
    <string>Afficher/Cacher le contrôle de flux</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSauvegarder_les_donn_es_acquises">
   <property name="text">
    <string>Sauvegarder les données acquises</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAttacher_D_tacher_une_fen_tre">
   <property name="text">
    <string>Attacher/Détacher une fenêtre</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionPlein_cran">
   <property name="text">
    <string>Plein écran</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Mdi</class>
   <extends>QFrame</extends>
   <header>View/mdi.h</header>
   <container>1</container>
   <slots>
    <signal>onResize()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonNext</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nextFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>-22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mdiAreaMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeMdiMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPrevious</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>previousFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderCurseur</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonactiveHandle</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>activeHandle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPlay</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>playPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mdiArea</sender>
   <signal>onResize()</signal>
   <receiver>MainWindow</receiver>
   <slot>resizeMdi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>-6</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAfficher_les_param_tres</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showHideDockParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVue_par_d_faut</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>enterInDefaultMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVue_Tabulation</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>enterInTabulationMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVue_Libre</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>enterInFreeMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAfficher_Cacher_le_contr_le_de_flux</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showHideDockStreamControl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAttacher_D_tacher_une_fen_tre</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>attachDetach()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlein_cran</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fullscreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeSource1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeSource2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nextFrame()</slot>
  <slot>fullscreen()</slot>
  <slot>changeMdiMode(int)</slot>
  <slot>previousFrame()</slot>
  <slot>sliderMoved(int)</slot>
  <slot>activeHandle()</slot>
  <slot>playPause()</slot>
  <slot>resizeMdi()</slot>
  <slot>showHideDockParameters()</slot>
  <slot>enterInDefaultMode()</slot>
  <slot>enterInTabulationMode()</slot>
  <slot>enterInFreeMode()</slot>
  <slot>showHideDockStreamControl()</slot>
  <slot>attachDetach()</slot>
  <slot>changeSource1()</slot>
  <slot>changeSource2()</slot>
 </slots>
</ui>
